src/Network/Network/help/Get-AzNetworkManagerEffectiveConnectivityConfigurationList.md

```diff
@@ -0,0 +1,151 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzNetworkManagerEffectiveConnectivityConfigurationList
+
+## SYNOPSIS
+Lists NetworkManager Effective Connectivity Configurations applied on a virtual networks.
+
+## SYNTAX
+
+```
+Get-AzNetworkManagerEffectiveConnectivityConfigurationList -VirtualNetworkName <String>
```

Comment:
I would prefer to keep as-is, as this command will always return a list of all effective configs Let me know what you think; it will never return a non-list response.